usr/src/cmd/bhyve/Makefile

```diff
@@ -64,6 +64,7 @@ SRCS =	acpi.c			\
 	pctestdev.c		\
 	pm.c			\
 	post.c			\
+	privileges.c		\
```

Comment:
how about namespacing the filename since all the internal functions seem to be namespaced ...

Reply:
I'm planning to just leave it as `privileges.[ch]`, and if the same file appears in the upstream freebsd source then we'll have to rename it then.